pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## What’s the focus of this PR
```

Comment:
Is it being added under the `.github` folder?

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

apparently it is an option if we want it hidden, should I move it?

Reply:
Ohhhh, I didn't know that there were several options for that. Cool!!

In my opinion, it's better to have it hidden as it's not part as the code itself. But feel free to put it wherever you think is best. I'll approve the PR anyway.

Reply:
Nice, let's move it then!